app/views/sessions/createsession.scala.html

```diff
@@ -127,6 +127,7 @@
         <div class="col-md-4"></div>
     </div>
 
+
     @feedback.feedbackformpreview()
 
-}
+}
```

Comment:
This symbol at the end of the line depicts that a file should end with a new empty line, make sure to follow it in all files. Also, reformat the file using intellij.